data/json/dreams.json

```diff
@@ -237,7 +247,11 @@
   },
   {
     "type": "dream",
-    "messages": [ "You dream of living deep inside a dark cave.", "You dream of being a primitive cave dweller." ],
+    "messages": [
+      "You dream of living deep inside a dark cave.",
+      "The Tommyknockers are our friends.  They warn us of danger.  We'll leave a little cake for them on this rock.",
```

Comment:
Google tells me this is a stephen king reference, so deos this assume that every character has read the novel? More importantly players that haven't read it (like me) will have no idea what this mean. Again the dreams we have currently are about being the thing you're becoming and I think sticking to that would be good.

Reply:
while King did write a book, tommyknockers are not his creation - tommyknockers are the ghosts of dead miners who warn those traveling under ground of danger (ie, mine collapse, pitfall) - https://www.legendsofamerica.com/gh-tommyknockers/

Reply:
Is it a common thing in new england? Still this feels  out of place in the mutant dreams.

Reply:
ok, I will remove it